draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -423,7 +423,11 @@ It is important that not every flow be prioritized; otherwise, the network devol
 
 Such a mechanism might be simple, for example, a cellular network might allow one flow from a subscriber to declare itself as important; other flows with that subscriber are denied attempts to prioritize themselves. However, the network cannot identify whether the prioritized flow is legitimate or malicious.
 
-Requirement: REQ-??: *need text here*
+Requirements:
+
+  REQ-SIGNAL-VALIDATION: The network/OS needs to ensure that the user/client signaling of priority is unique for every type of traffic within the same flow, thereby avoiding prioritizing of all the streams/traffic the same way.
```

Comment:
If the client/user signals metadata A to be highest priority and then signals metadata B to be highest priority, do we just remove A (assign it default priority) and make B higher priority or discard the whole priority table and reset all the traffic to default values?